src/test/java/com/google/neighborgood/UserInfoServletTest.java

```diff
@@ -206,14 +205,13 @@ public void normalMultipleInputTest() throws IOException {
 
     // After sending the second POST request, there should be still three entities in the datastore
     assertEquals(3, ds.prepare(new Query("UserInfo")).countEntities(withLimit(10)));
-    results =
-        ds.prepare(
-            new Query("UserInfo")
-                .setFilter(new FilterPredicate("userId", FilterOperator.EQUAL, "1234567890")));
-    entity = results.asSingleEntity();
 
-    // The entity can't be null
-    assertNotNull(entity);
+    try {
+      entity = ds.get(userEntityKey);
+    } catch (EntityNotFoundException e) {
+      // The entity can't be null
+      assertNotNull(entity);
```

Comment:
Same problem here. Maybe the assertNotNull statement should be placed out of the catch clause

Reply:
Got it, I can move it to outside of the catch statement.